website/docs/core-concepts/connecting-to-data-sources/connecting-to-databases.md

```diff
@@ -5,59 +5,75 @@ description: >-
 ---
 
 # Databases
- You can connect to, and run queries to read and write data directly from the
-  Appsmith editor.
 
-## Supported Databases
+Databases are an essential part of many applications and are used to store and manage data. They provide a structured way to organize, retrieve, and manipulate data, and are designed to be efficient and reliable. You can run queries to read and write data directly from the Appsmith editor. 
 
-* [Amazon S3 (also Upcloud, Digital Ocean Spaces, Wasabi, DreamObjects)](../../reference/datasources/querying-amazon-s3.md)
-* [ArangoDB](../../reference/datasources/querying-arango-db.md)
-* [DynamoDB](../../reference/datasources/querying-dynamodb.md)
-* [ElasticSearch](../../reference/datasources/querying-elasticsearch.md)
-* [Firestore](../../reference/datasources/querying-firestore.md)
-* [Google Sheets](../../reference/datasources/querying-google-sheets.md)
-* [MongoDB](../../reference/datasources/querying-mongodb/)
-* [MS SQL](../../reference/datasources/querying-mssql.md)
-* [MySQL](../../reference/datasources/querying-mysql.md)
-* [PostgreSQL](../../reference/datasources/querying-postgres.md)
-* [Redis](../../reference/datasources/querying-redis.md)
-* [Redshift](../../reference/datasources/querying-redshift.md)
-* [Snowflake](../../reference/datasources/querying-snowflake-db.md)
-* [SMTP](../../reference/datasources/using-smtp.md)
+### Connecting to a database
 
-## Security
+<VideoEmbed host="youtube" videoId="sJIxtXInV14" title="How to connect to a datasource" caption="How to connect to a datasource | Example"/>
 
-Appsmith safely encrypts all your database credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Since Appsmith is an open-source framework, you can [deploy it on-premise](../../getting-started/setup/), and audit it to ensure none of your data leaves your VPC.
 
-## Connecting to a Database
+:::tip
```

Comment:
Keep this information as Important info - info callout and name it important . Also mention clearly that it is for self hosted instances.

Also sharing a useful link on how to decide whether a callout should be tip, info, note , etc.- https://paligo.net/docs/en/admonitions--notes,-warnings,-tips,-etc--.html

Shorten the callout. Its too verbose. and remove the "how to whitelist appsmith on AWS" part . This should be cross-reference on the AWS datasource page

Reply:
done